templates/GlobalToolbarProfile.ss

```diff
@@ -91,6 +104,9 @@ document.addEventListener('DOMContentLoaded', function(){
 		if('parentIFrame' in window) {
 			parentIFrame.size(60, w);
 			window.clearInterval(interval);
+
+			// Apply iframe ID to body class
```

Comment:
Could you please elaborate a bit more in this comment to explain why it's required? Feel free to use `<%-- %>` SilverStripe template comments if you don't want it to be rendered on the frontend